plugins/micromega/numCompat.ml

```diff
@@ -304,7 +304,7 @@ module Z_mix : ZArith with type t = Z_old.t * Z_new.t = struct
     ((r_old1, r_new1), (r_old2, r_new2))
 
   let zero = (Z_old.zero, Z_new.zero)
-  let one = (Z_old.zero, Z_new.zero)
+  let one = (Z_old.one, Z_new.one)
```

Comment:
:face_with_head_bandage: :facepalm: :man_facepalming: :woman_facepalming: :woman_astronaut:

Reply:
Updated, and added some more debug code.